bang/magicollections.py

```diff
@@ -2657,6 +2669,17 @@ class ListView(MagiCollection.ListView):
         # not with 4 to a row
         show_relevant_fields_on_ordering = False
 
+        alt_views = MagiCollection.ListView.alt_views + [
+            ('chibis', { 'verbose_name': _('Chibi'), 'per_line': 2 }),
+        ]
+
+        def get_queryset(self, queryset, parameters, request):
+            queryset = super(CostumeCollection.ListView, self).get_queryset(queryset, parameters, request)
+            if parameters.get('view') == 'chibis':
+                return queryset.filter(owned_chibis__isnull=False).distinct()
+            else:
+                return queryset.filter(model_pkg__isnull=False).exclude(model_pkg='')
```

Comment:
Does that mean there might be a chance that some chibis will only be available in the alt chibis view? Would that be a common case?
If that's so we may consider adding a link to the chibis view in the nav bar because not a lot of people notice the view switcher at all.
If there are no such cases atm then that's fine!

Reply:
there shouldn't be, as ingame every set of chibis has a live2d model. we just have to maintain them properly